groovy-modifications/README.md

```diff
@@ -0,0 +1,27 @@
+# Tutorial on how to use network modifications methods in Groovy
+
+In this tutorial, you will find two scripts explaining how to add a load and a line with the topology to a network.
+
+The network element is added to the network and the switches are also created to connect the element 
+to a given busbar section with a closed disconnector. Open switches are also added to every parallel busbar sections.
+
+Other similar functions are available to add every kind of injections and branches.
+
+On all the scripts, a node/breaker is imported. The single line diagram of voltage levels S1VL2 and S2VL1 of this network are:
+![S1VL2](.github/nb_network_s1vl2.png "S1VL2")
+![S2VL1](.github/nb_network_s2vl1.png "S2VL1")
```

Comment:
Maybe display the networks in the points below, directly in the examples in which they are used (before/after) for more clarity